src/lib/rule2C.ts

```diff
@@ -0,0 +1,35 @@
+import {Context, getDefaultContext} from './context';
+import {rule2E} from './rule2E';
+
+/**
+ * Implementation for rule 2C
+ * @param node - node whose text alternative is being computed
+ * @param context - information relevant to the computation of node's text alternative
+ * @return text alternative for 'node' if rule 2C accepts 'node', null otherwise.
+ */
+export function rule2C(
+  node: Node,
+  context: Context = getDefaultContext()
```

Comment:
you should be able to drop the type when you have a default argument of the right type